<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>offensiveAttributesWidget</class>
 <widget class="QWidget" name="offensiveAttributesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>108</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>offensiveAttributesWidget</string>
  </property>
  <widget class="QLabel" name="offensiveLabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>0</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Offensive</string>
   </property>
  </widget>
  <widget class="QLabel" name="shotThreeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>55</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Shot 3pt</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>0</y>
     <width>206</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="shotMediumLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>55</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Shot Med</string>
   </property>
  </widget>
  <widget class="QLabel" name="shotCloseLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>55</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Shot Cls</string>
   </property>
  </widget>
  <widget class="QLabel" name="shootInTrafficLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>55</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>In Traffic</string>
   </property>
  </widget>
  <widget class="QLabel" name="shotInTrafficAttributeLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>20</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>99</string>
   </property>
  </widget>
  <widget class="QLabel" name="shotCloseAttributeLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>20</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>99</string>
   </property>
  </widget>
  <widget class="QLabel" name="shotMediumAttributeLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>20</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>99</string>
   </property>
  </widget>
  <widget class="QLabel" name="shotThreeAttributeLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>20</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>99</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>36</y>
     <width>80</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>56</y>
     <width>80</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>76</y>
     <width>80</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>250</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="shootOffDribbleLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>55</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Off Dribble</string>
   </property>
  </widget>
  <widget class="QLabel" name="postHookLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>55</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>PHook</string>
   </property>
  </widget>
  <widget class="QLabel" name="postFadeawayAttributeLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>20</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>99</string>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>76</y>
     <width>80</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="postHookAttributeLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>20</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>99</string>
   </property>
  </widget>
  <widget class="QLabel" name="shootOffDribbleAttributeLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>20</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>99</string>
   </property>
  </widget>
  <widget class="QLabel" name="lowPostOffenseLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>55</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>LP Offense</string>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>36</y>
     <width>80</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="postFadeawayLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>55</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>PFadeaway</string>
   </property>
  </widget>
  <widget class="QLabel" name="lowPostOffenseAttributeLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>20</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>99</string>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>56</y>
     <width>80</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="consistencyAttributeLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>20</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>99</string>
   </property>
  </widget>
  <widget class="QLabel" name="consistencyLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>55</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Consistent</string>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>36</y>
     <width>80</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_10">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>15</y>
     <width>20</width>
     <height>81</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_11">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>15</y>
     <width>20</width>
     <height>81</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
